ToolsForMaintainersOfTheProjectTemplate/Scintilla_iface_synchronizer/cs.py

```diff
@@ -35,12 +35,12 @@ def isTypeUnsupported(t):
 	if t in ["formatrange"]: return True
 	return False
 
-def translateType(t):
+def translateType(t:str):
 	if t == "cells": return "Cells"
 	if t == "colour": return "Colour"
-	if t == "line": return "int"
+	if t == "line": return "long"
```

Comment:
line and position types need to be atleast 64-bit wide which can accommodate both 32-bit data in x86 execution and 64-bit data in x64 execution.

IntPtr could have been used as suggested, but that complicates things with arithmetic operations.

Reply:
is this the correct assumption if you build a 32bit and a64 bit version

Reply:
you are right, there is a potential issue for **set** operations, if we allow the long to be used and user supplies a long value but the execution env is x86. This might result in overflow/truncation.

I was just focusing on **get** operation where the system returned type can fit on the target container for both x86 and x64.

So we are again left with only option of using **IntPtr** to make it work for both environment.

Reply:
I have suggested alternative fix for this: see my reply [below](https://github.com/kbilsted/NotepadPlusPlusPluginPack.Net/pull/75#issuecomment-849363118)